examples/tokens/token_airdrop_claim_auto.py

```
@@ -238,7 +213,7 @@ def log_fungible(
         for token_id in token_ids:
             print(f"    {token_id}: {balances.get(token_id, 0)}")
 
-    def log_nfts(account_id: AccountId, nft_ids: Iterable[NftId]):
+    def log_nfts(account_id: AccountId, nft_ids: Iterable[TokenId]):
```

Ruff (0.14.10)

Missing return type annotation for private function `log_nfts`

Add return type annotation: `None`

(ANN202)